apps/evm/src/clients/api/queries/useGetPools/getPools/formatOutput/formatDistributions/index.ts

```diff
@@ -103,10 +107,12 @@ export const formatDistributions = ({
 
         borrowDistributions.push(
           formatRewardDistribution({
+            rewardType,
             rewardToken,
             rewardTokenPriceDollars,
             dailyDistributedRewardTokens,
             balanceDollars: borrowBalanceDollars,
+            description: rewardDetails?.description || '',
```

Comment:
```suggestion
            description: rewardDetails?.description,
```

This makes it easier for the UI to handle cases where there's no description associated with the reward.